style.css

```diff
@@ -60,11 +67,46 @@ body {
   }
 }
 
-.menu-button {
+.humberger {
   border: none;
   background-color: #f7f7f9;
 }
 
+#mobile_menu {
+  position: fixed;
+  display: none;
+  flex-direction: column;
+  justify-content: start;
+  background-color: rgba(96, 112, 255, 0.9);
+  gap: 8px;
+  font-size: larger;
+  list-style-type: none;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  margin: 0;
+  z-index: 99;
+  backdrop-filter: blur(5px);
+}
```

Comment:
[OPTIONAL] Consider using `mix-blend-mode: multiply` for the overlay blur effect since that gives you the best view. :wink: